wear/src/main/java/chat/rocket/android/util/String.kt

```diff
@@ -0,0 +1,21 @@
+package chat.rocket.android.util
+
+fun String.removeTrailingSlash(): String {
+    return if (isNotEmpty() && this[length - 1] == '/') {
+        this.substring(0, length - 1)
+    } else {
+        this
+    }
+}
+
+fun String.avatarUrl(
```

Comment:
This is duplicate from the existing extension function defined in the `app` module. Please, remove.